Makefile

```diff
@@ -56,6 +58,23 @@ deploy-chart: ## Deploys helm chart and index file to S3 path specified by HELM_
 	aws s3 cp packaging/helm/aws-servicebroker-*.tgz $(S3URI) --acl $(ACL) $(PROFILE) && \
 	aws s3 cp packaging/helm/index.yaml $(S3URI) --acl $(ACL) $(PROFILE)
 
+release: ## Package and deploy requirements for a release
+	make clean && \
+	mkdir -p release/$(VERSION) && \
+	make build && \
+	mv ./servicebroker release/$(version)/aws-servicebroker-$(VERSION)-OSX && \
```

Comment:
There are three references to `$(version)`. Do those need to be changed to `$(VERSION)`?